spring-boot-project/spring-boot-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -678,42 +677,42 @@ application:
 	........ Started Example in 2.222 seconds (JVM running for 6.514)
 ----
 
-If you open a web browser to http://localhost:8080 you should see the following output:
+If you open a web browser to http://localhost:8080, you should see the following output:
 
 [indent=0]
 ----
 	Hello World!
 ----
 
-To gracefully exit the application hit `ctrl-c`.
+To gracefully exit the application, press `ctrl-c`.
 
 
 
 [[getting-started-first-application-executable-jar]]
-=== Creating an executable jar
-Let's finish our example by creating a completely self-contained executable jar file that
+=== Creating an Executable Jar
+We finish our example by creating a completely self-contained executable jar file that
 we could run in production. Executable jars (sometimes called "`fat jars`") are archives
 containing your compiled classes along with all of the jar dependencies that your code
 needs to run.
 
 .Executable jars and Java
 ****
-Java does not provide any standard way to load nested jar files (i.e. jar files that are
+Java does not provide a standard way to load nested jar files (jar files that are
 themselves contained within a jar). This can be problematic if you are looking to
 distribute a self-contained application.
 
-To solve this problem, many developers use "`uber`" jars. An uber jar simply packages
-all classes, from all jars, into a single archive. The problem with this approach is that
-it becomes hard to see which libraries you are actually using in your application. It can
+To solve this problem, many developers use "`uber`" jars. An uber jar packages
+all the classes from all the contained jars into a single archive. The problem with this approach is that
```

Comment:
Contained isn't quite right here. Perhaps just "all the jars" or, more accurately, "all the application's dependencies' jars"

Reply:
I find possessives of possessives to be hard to follow, so I changed the sentence to "An uber jar packages all the classes from all the application's dependencies into a single archive."

Does that work for you?